.github/workflows/test-podman-binary-n2sn.yml

```
@@ -0,0 +1,101 @@
+name : test_podman_binary_n2sn
+
+on:
+  pull_request:
+
+jobs:
+  generate_conda_packd_envs:
+    name: n2sn conda-pack'd env with Python build using podman as the docker binary ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7"]
+      fail-fast: false
+
+    steps:
+      - name: Set env.REPOSITORY_NAME  # just the repo, as opposed to org/repo
+        shell: bash -l {0}
+        run: |
+          export REPOSITORY_NAME=${GITHUB_REPOSITORY#*/}
+          echo "REPOSITORY_NAME=${REPOSITORY_NAME}" >> $GITHUB_ENV
+
+      - name: Set secrets
+        shell: bash -l {0}
+        run: |
+          export ZENODO_ACCESS_TOKEN=${{ secrets.ZENODO_ACCESS_TOKEN }}
+          echo "ZENODO_ACCESS_TOKEN=${ZENODO_ACCESS_TOKEN}" >> $GITHUB_ENV
+
+      - name: Set non-secret environment variables
+        shell: bash -l {0}
+        run: |
+          # ZENODO_OWNER_ID is an integer number corresponding to the sandbox
+          # account with the token above, that is used to upload files, and to
+          # search by the owner id.
+          export ZENODO_OWNER_ID="40261"  # mrakitin's owner id on sandbox.zenodo.org
+          export DOCKER_BINARY="podman"
+          echo "ZENODO_OWNER_ID=${ZENODO_OWNER_ID}" >> $GITHUB_ENV
+          echo "DOCKER_BINARY=${DOCKER_BINARY}" >> $GITHUB_ENV
```

Suggesting combining the parameters in groups, so that it's easier to remove/duplicate a block if needed.
```suggestion

          export DOCKER_BINARY="podman"
          echo "DOCKER_BINARY=${DOCKER_BINARY}" >> $GITHUB_ENV
```